project_name/settings/base.py

```diff
@@ -183,15 +183,45 @@
     }
 }
 
-if "AWS_STORAGE_BUCKET_NAME" in os.environ:
+
+# Check for either DEFAULT_* (Divio) or AWS_* environment variables
+
+# AWS_STORAGE_BUCKET_NAME: The name of the S3 bucket to use for storage.
+AWS_STORAGE_BUCKET_NAME = os.environ.get("DEFAULT_STORAGE_BUCKET", "") or os.environ.get("AWS_STORAGE_BUCKET_NAME", "")
+
+# AWS_ACCESS_KEY_ID: The access key ID for authenticating with AWS S3.
+AWS_ACCESS_KEY_ID = os.environ.get("DEFAULT_STORAGE_ACCESS_KEY_ID", "") or os.environ.get("AWS_ACCESS_KEY_ID", "")
+
+# AWS_SECRET_ACCESS_KEY: The secret access key for authenticating with AWS S3.
+AWS_SECRET_ACCESS_KEY = os.environ.get("DEFAULT_STORAGE_SECRET_ACCESS_KEY", "") or os.environ.get("AWS_SECRET_ACCESS_KEY", "")
+
+# We generally use this setting in the production to put the S3 bucket
+# behind a CDN using a custom domain, e.g. media.llamasavers.com.
+# https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#cloudfront
+AWS_S3_CUSTOM_DOMAIN = os.environ.get("DEFAULT_STORAGE_CUSTOM_DOMAIN", "") or os.environ.get("AWS_S3_CUSTOM_DOMAIN", "")
```

Comment:
Issue: I believe many of these variables need to be unset rather than set to an empty string, hence the existence checks below. 

I think the bucket name check should stay (supporting 2 variables, of course), and the rest should probably stay inside the `if`.

Reply:
@RealOrangeOne Would it be possible for you to apply the changes you're suggesting? That would also help streamline the back-and-forth. We're fine with it as long as it works as intended.

Reply:
I've pushed https://github.com/torchbox/wagtail-news-template/pull/37/commits/d7dfb34f0cafd9551a59438719e6a49811cd4670 to clean this up a little.